hw/top_earlgrey/data/ip/chip_usbdev_testplan.hjson

```diff
@@ -162,7 +162,7 @@
       stage: V2
       si_stage: SV3
       lc_states: ["TEST_UNLOCKED", "DEV", "PROD", "PROD_END", "RMA"]
-      tests: []
+      tests: ["chip_sw_usbdev_config_host"]
       bazel: []
```

Comment:
should a bazel test be added as well?

Reply:
Yes, since this is one of the tests that does not require special host-side software, just the device stack.
I've added a separate commit to include bazel targets for those tests that exist and do not require host-side support, just cabling to an active host.